config.toml

```diff
@@ -3,12 +3,12 @@
 # Update it from https://api.github.com/meta
 allowed-cidr = [
         "127.0.0.1/24",
-        "207.97.227.253/32",
-        "50.57.128.197/32",
-        "108.171.174.178/32",
-        "50.57.231.61/32",
-        "204.232.175.64/27",
         "192.30.252.0/22",
+        "185.199.108.0/22",
+        "140.82.112.0/20",
+        "143.55.64.0/20",
+        "2a0a:a440::/29",
+        "2606:50c0::/32"
```

Comment:
New GitHub URLs were in https://api.github.com/meta. But it is not used right now. Should I remove it completely?

Reply:
We  can remove... btw. we can also remove toml dependency.

This was an experiment to try toml ... but I think that for technical usage yaml is more popular and for simple things .INI is better 

so, for this project I think that is best to just use .INI